kuka_mobile_robot_support/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.5)
+project(kuka_mobile_robot_support)
+
+find_package(ament_cmake REQUIRED)
+
+install(
+  DIRECTORY config gazebo launch ros2_control urdf meshes
```

Comment:
please follow the structure of the other support packages, where the ros2_control description is also in the `urdf` folder